src/reflect/scala/reflect/internal/transform/UnCurry.scala

```diff
@@ -77,6 +101,60 @@ trait UnCurry {
     }
   }
 
+  private def varargForwarderSym(currentClass: Symbol, origSym: Symbol, newInfo: Type): Symbol = {
+    val forwSym = currentClass.newMethod(origSym.name.toTermName, origSym.pos, VARARGS | SYNTHETIC | origSym.flags & ~DEFERRED)
```

Comment:
note that this also includes the fix for SI-10059 / https://github.com/scala/scala/pull/5534 which was not previously backported to 2.11.x. maybe add its test?